modules/core/src/main/java/org/locationtech/jts/geom/Coordinate.java

```diff
@@ -17,6 +17,8 @@
 import org.locationtech.jts.util.Assert;
 import org.locationtech.jts.util.NumberUtil;
 
+import javax.xml.bind.annotation.XmlTransient;
```

Comment:
This makes  JTS depend on JAXB / JAX-RI which is a JEE module as of Java 11 (it is still part of the Java 8 API though). Not sure that is a good idea.

Reply:
@mprins Good info.  Do you have any insight for alternatives to addressing the issue that @seanrl has raised here: https://github.com/locationtech/jts/issues/711 ?

Reply:
The proper solution would be to change the API of Coordinate and use a JavaBean pattern only (having only getters/setters for x,y,z); or alternatively a class member pattern only (having public fields/members only for x,y,z), or perhaps changing the visibility of x,y,z to les than `public`, not sure if eg `protected` would suffice. (NB adding this annotation to these public fields strictly incurs an API change as well)

The problem is that JAXB gets confused because it finds 2 ways to serialize x,y,z to XML and has no way to choose; the annotation exists to work around poor (or in case of JTS historic) design. `Coordinate` (or anything else in JTS) was ever designed to be an `@XMLType`. The workaround outside JTS is probably to use an `@XmlJavaTypeAdapter` I think, or perhap setting up the `@XmlAccessorType` in the container class.

personally I'd go for the bean pattern, always.

Reply:
I agree, the correct long-term solution is to make the x, y, and z fields private.  Of course, the XmlTransient annotation is safe as long as this is Java 8 based, so perhaps the low-hanging fruit here is to use the XmlTransient for now and deprecate the public fields, opting to switch to the JavaBean pattern at such a time when we migrate off of Java 8.

Reply:
There are many downstream projects relying on JTS that use Java 11 and do not have a JAXB dependency. 
IMO introducing the JAXB dependency + a public API change -which adding the annotation is- is worse than changing the API to follow the JavaBean pattern.

Reply:
@mprins That's a good point.  I'll close the PR, and just keep making my own internal releases until the JavaBean switch